cmd/base.go

```diff
@@ -249,6 +249,10 @@ func EnsureCLIFlags(cmd *cobra.Command, appCFG *bc.Config) error {
 			elem = reflect.ValueOf(&appCFG.Beacon).Elem()
 			assignFields(ff, elem, name)
 
+			ff = reflect.TypeOf(appCFG.POET)
+			elem = reflect.ValueOf(&appCFG.POET).Elem()
+			assignFields(ff, elem, name)
```

Comment:
@dshulyak i may have given you the wrong namespace yesterday. but this is why it didn't work yesterday.
the flags set through systest did not take effect.

sigh... the config/flag setting is quite convoluted...